test/data/repositories/rhel-86.json

```diff
@@ -2,53 +2,53 @@
   "x86_64": [
     {
       "name": "baseos",
-      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-baseos-n8.6-20210921/",
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-baseos-n8.6-20211116/",
```

Comment:
seems like this snapshot still fails? https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1791561513#L2698 @achilleas-k

Reply:
seems like it's building on centos..... why? :thinking:

Reply:
updated centos too